.whitesource

```diff
@@ -1,3 +1,10 @@
 {
-  "settingsInheritedFrom": "SmartBear/whitesource-config@main"
```

Comment:
@IlyaAvdeev Please keep this line so the rest of the configuration remains consistent. Any other value specified in this file directly will override the value specified in the default config.